control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -232,7 +233,7 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, cg *kafkainternals.Consum
 	cg.Spec.Replicas = pointer.Int32(0)
 	err := r.schedule(ctx, cg) //de-schedule placements
 
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "no scheduler found") {
```

Comment:
instead of contains on error message I think using `errors.Is(...)` with a custom error is more idiomatic and robust

Reply:
Can we also add a test in consumergroup_test.go